src/chatClient.cpp

```diff
@@ -3831,6 +3838,7 @@ bool GroupChatRoom::syncWithApi(const mega::MegaTextChat& chat)
         onArchivedChanged(mIsArchived);
     }
 
+    mAutoJoiningCompleted = true;
```

Comment:
I think this is not the right place, since it will set this value, in example, when the title changes.
The flag should be set exclusively when the own user has rejoined the room.

Reply:
Fixed